api/src/main/java/org/openmrs/module/metrics/builder/impl/JmxReportBuilderImpl.java

```diff
@@ -0,0 +1,31 @@
+package org.openmrs.module.metrics.builder.impl;
+
+import com.codahale.metrics.MetricRegistry;
+import com.codahale.metrics.jmx.JmxReporter;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.openmrs.module.metrics.builder.JmxReportBuilder;
+
+public class JmxReportBuilderImpl implements JmxReportBuilder {
+	
+	protected final Log log = LogFactory.getLog(this.getClass());
+	
+	private static MetricRegistry metricRegistry = new MetricRegistry();
```

Comment:
Do we not want this to be an instance field and possibly injectable? Non-final static variables are generally an kind of anti-pattern unless there's a good justification for them (among other things, they are hard to test and tend to create global state).

Reply:
Actually, looking a bit closer, I don't understand why we have this field at all, since we seem to be using the registry passed in to `start()` and `stop()`?